src/routes/participant/schedule/[rehearsalId]/+page.ts

```diff
@@ -16,13 +15,6 @@ import type { PageLoad } from './$types'
 
 export const ssr = false
 
-type playsSong = {
```

Comment:
right, want deze type had ik niet toevallig niet meer nodig in mn refactor branch ofzo :kek:

Reply:
? wut